src/coreclr/tools/Common/JitInterface/ThunkGenerator/InstructionSetDesc.txt

```diff
@@ -81,9 +83,7 @@ instructionset64bit,X86   ,LZCNT
 instructionset64bit,X86   ,PCLMULQDQ
 instructionset64bit,X86   ,POPCNT
 instructionset64bit,X86   ,AVXVNNI
-instructionset64bit,X86   ,MOVBE
 instructionset64bit,X86   ,X86Serialize
-instructionset64bit,X86   ,EVEX
```

Comment:
Neither of these 64-bit ISA variants is used for anything.

MOVBE is used opportunistically for the special GT_BSWAP node, which is only used for 64-bit values on 64-bit platforms

https://github.com/dotnet/runtime/blob/35d27803b83bbd46d3cfe42aa58f46f585c0c6ae/src/coreclr/jit/importercalls.cpp#L4642-L4646

EVEX is a virtual ISA that represents either AVX10v1 or AVX-512 and is used for opportunistic EVEX encoding and kmask operations.